src/component/calculator.js

```diff
@@ -1,52 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './calculator.css';
+import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
+function Calculator() {
```

Comment:
[OPTIONAL] Since you use ES6, it would be nicer to use arrow functions instead of the `function` word.

Reply:
arrow function being used now